src/OrleansTestingHost/TestCluster.cs

```diff
@@ -585,6 +586,7 @@ private Silo LoadSiloInNewAppDomain(string siloName, Silo.SiloType type, Cluster
                 new object[] { });
 
             appDomain.UnhandledException += ReportUnobservedException;
+            appDomain.DoCallBack(RegisterPerfCountersTelemetryConsumer);
```

Comment:
Just to clarify... This hack here and on TestSiloHost is something me and @jdom agreed to leave for now since we can't configure properly via code config the telemetry consumer inside the test cluster/silo AppDomain from the xunit tester.

I'll revisit it later once I get the next task for the new config and silo host model and this will (finally) get out here.